### PR TITLE
Enable monthly dependency updates for UI.

### PR DESCRIPTION
**Pull request by michelle192837**
Fixes https://github.com/kubernetes-sigs/testgrid/issues/9

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  - package-ecosystem: "pnpm" # See documentation for possible values
+    directory: "/web" # Location of package manifests
+    schedule:
+      interval: "monthly"
+
```
Comment (ameukam) [inline .github/dependabot.yml:12]:
```suggestion
version: 2
updates:
  - package-ecosystem: "pnpm" # See documentation for possible values
    directory: "/web" # Location of package manifests
    schedule:
      interval: "monthly"
		labels:
			- "ok-to-test"
```
Dependabot is not part of the org. we make sure the tests are automatically triggered for any PR.

**Comment by michelle192837** [comment]
From running the test locally, it seems like sometimes `npm test` or `npm run test` will work, but I can reproduce the error reliably with `rm -rf node_modules/; npm install && npm test`. `pnpm test` doesn't seem to have the same issue, updating the job in test-infra to try that out.

File: .github/dependabot.yml (hunk 1 above)
Comment (michelle192837) [reply .github/dependabot.yml]: Oh, neat pattern! Added, I think this should be fine (but we should still manually LGTM imo).

**Comment by michelle192837** [comment]
Ah. Okay, one more fix.

**Comment by ameukam** [comment]
> Ah. Okay, one more fix.

we probably want to include `pnpm` in the dependencies list.

**Comment by michelle192837** [comment]
> > Ah. Okay, one more fix.
> 
> we probably want to include `pnpm` in the dependencies list.

Probably, but we need it globally installed to run on the command line I believe.

Added https://github.com/kubernetes/test-infra/pull/33853

**Pull request merged**